### PR TITLE
Tolerance relaxation for convertstate test

### PR DESCRIPTION
**Pull request by clouden90**
## Description

Relax TOL for convertstate test

Is a change of answers expected from this PR? No

## Testing

How were these changes tested? ctest
What compilers / HPCs was it tested with? gnu/intel18/intel19
Are the changes covered by ctests? Yes

### CODE REVIEW

**Comment by guillaumevernieres** [comment]
@clouden90 : I need a bit more context! 1e-5 seems quite large and I don't remember seeing the issue before. What compiler/hpc are you using?

**Comment by clouden90** [comment]
> @clouden90 : I need a bit more context! 1e-5 seems quite large and I don't remember seeing the issue before. What compiler/hpc are you using?


No problem, G. Currently the results in testref/convertstate.test are based on GNU Fortran (GCC) 9.3.0. If you try to build SOCA on Hera/Orion with Intel18/Intel19 and run ctest for test_soca_convertstate, you will get the following message (screenshot is attached, too):
47: Run: Finishing oops::ConvertState<SOCA> with status = 0
47:
47: ===============================================================================
47: Running compare script
47: ===============================================================================
47: Float mismatch at line 13: 5.978337  not equal to  5.978343  with max relative difference  1e-12  Actual relative difference =  1.0036234150612578e-06
47: Failed in the COMPARE step. Error code: 1

Basically is shows the mean of tocn (5.978337) is not equal to the one in ref (5.978343). All the others are good.

image

**Review by guillaumevernieres — APPROVED** [review]
:+1:
